csharp-algorithms/QuickSort/QuickSortTests/QuickSortUnitTests.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using QuickSort;
+using System;
+using System.Diagnostics;
+
+namespace QuickSortTests
+{
+    [TestClass]
+    public class QuickSortUnitTests
```

Comment:
This is a great oportunity to test for StackOverflowException you've mentioned in the article. You can even mention that they can check it out in the code if curious.